### ISSUE TITLE
Text in voice channels crashing

### ISSUE THREAD

**Issue by WilsontheWolf** [abalabahaha/eris#1381]
Discord recently rolled out text in voice channels (see https://support.discord.com/hc/en-us/articles/4412085582359)
When a message is sent in one of these it crashes Eris.
```
node:events:368
      throw er; // Unhandled 'error' event
      ^

TypeError: Cannot read properties of undefined (reading 'add')
    at Shard.wsEvent (/home/shorty/projects/DoTheMario/node_modules/eris/lib/gateway/Shard.js:853:65)
    at Shard.onPacket (/home/shorty/projects/DoTheMario/node_modules/eris/lib/gateway/Shard.js:402:26)
    at Shard._onWSMessage (/home/shorty/projects/DoTheMario/node_modules/eris/lib/gateway/Shard.js:2446:29)
    at WebSocket.emit (node:events:390:28)
    at Receiver.receiverOnMessage (/home/shorty/projects/DoTheMario/node_modules/ws/lib/websocket.js:1169:20)
    at Receiver.emit (node:events:390:28)
    at Receiver.dataMessage (/home/shorty/projects/DoTheMario/node_modules/ws/lib/receiver.js:528:14)
    at Receiver.getData (/home/shorty/projects/DoTheMario/node_modules/ws/lib/receiver.js:446:17)
    at Receiver.startLoop (/home/shorty/projects/DoTheMario/node_modules/ws/lib/receiver.js:148:22)
    at Receiver._write (/home/shorty/projects/DoTheMario/node_modules/ws/lib/receiver.js:83:10)
Emitted 'error' event on Client instance at:
    at Shard.emit (/home/shorty/projects/DoTheMario/node_modules/eris/lib/gateway/Shard.js:242:26)
    at Shard._onWSMessage (/home/shorty/projects/DoTheMario/node_modules/eris/lib/gateway/Shard.js:2449:18)
    at WebSocket.emit (node:events:390:28)
    [... lines matching original stack trace ...]
    at Receiver._write (/home/shorty/projects/DoTheMario/node_modules/ws/lib/receiver.js:83:10)
    at writeOrBuffer (node:internal/streams/writable:389:12)
```

**Comment by DonovanDMC** [comment]
Known issue, open pr to fix this: #1298

**Comment by WilsontheWolf** [comment]
Ok sounds good. It seems I can handle the error to stop the crash.
```js
client.on('error', (e) => {
    console.error(e);
});
```
